chartsxblock/static/js/src/chartsxblock-studio.js

```diff
@@ -142,12 +142,27 @@ function ChartTable(targetDiv, inputJSON){
 
     this.draw = function() {
     // Draws the table in a div
+        var delete_column_row = $("<tr>"),
+            delete_column = '';
+
+        for(var i = 0; i < this.columns; i++) {
```

Comment:
Ne sviđa mi se sasvim ova dodatna petlja, ili radije, ne sviđa mi se gdje je pozicionirana. Hajde pogledaj način da je ubaciš negdje u petlji ispod - na malo logički smislenije mjesto.